components/Tools/FileConverter/PackagesVersions.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Why is there json schema here?

Reply:
The versions schemas stayed in the front, because it's the front that knows which packages are used in each tool